uqcsbot/scripts/dice.py

```diff
@@ -0,0 +1,31 @@
+from random import choice
+from uqcsbot import bot, Command
+
+def is_number(message):
+    '''
```

Comment:
PEP8 suggests triple double quotes for docstrings.

Reply:
This was copied from `mock.py`.